homeassistant/components/ssdp/__init__.py

```diff
@@ -213,8 +218,8 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         UPNP_SERVER: server,
     }
 
-    hass.create_task(scanner.async_start())
-    hass.create_task(server.async_start())
+    await hass.async_create_task(scanner.async_start())
+    await hass.async_create_task(server.async_start())
```

Comment:
Does this need to be wrapped in a task, or can it be awaited ?

Reply:
I think it can be awaited. This came from https://github.com/home-assistant/core/blame/d12cbab6c4c61c48bc72ebbc3fe9483db107023c/homeassistant/components/ssdp/__init__.py#L175. Moved to `hass.async_create_task()` otherwise unit tests might not always start the server.

I've changed it to an await.